lighthouse-core/gather/driver.js

```diff
@@ -464,8 +464,6 @@ class Driver {
         });
     }
 
-    log.verbose('Driver', `Installing longtask listener for CPU idle.`);
-    this.evaluateScriptOnLoad(`(${installPerformanceObserver.toString()})()`);
```

Comment:
doesn't this mean we can also get rid of `markAsResolvable` since registration and polling are separated?